monkey/monkey_island/cc/models/monkey.py

```diff
@@ -42,6 +43,9 @@ class Monkey(Document):
     ttl_ref = ReferenceField(MonkeyTtl)
     tunnel = ReferenceField("self")
     command_control_channel = EmbeddedDocumentField(CommandControlChannel)
+
+    # Environment related fields
+    environment = StringField(default=environment_names.UNKNOWN, choices=environment_names.ALL_ENV_NAMES)
```

Comment:
is it used?

Reply:
Will be used in #519, that's the whole reason I did this PR :)